templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Login</h2>
+  {% if form.errors %}
+    <p style="font-weight: bold; color: red;">
+      Invalid credentials. Try again!
+    </p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">

```

Comment:
The form action is set to the `login` URL, which is correct. Ensure that the `login` URL is correctly configured in your authentication URLs.